dockstore-webservice/src/test/java/io/dockstore/webservice/helpers/EntryVersionHelperTest.java

```diff
@@ -21,4 +29,53 @@ public void removeWorkingDirectory() {
         Assert.assertEquals(".dockstore.yml", EntryVersionHelper.removeWorkingDirectory("./.dockstore.yml", ".dockstore.yml"));
         Assert.assertEquals(".dockstore.yml", EntryVersionHelper.removeWorkingDirectory("././.dockstore.yml", ".dockstore.yml"));
     }
+
+    /**
+     * Tests that there's no exceptions when create zip file with null content or empty content
+     * Does not test the possible IOException from zipOutputStream.putNextEntry and zipOutputStream.closeEntry
+     * @throws IOException
+     */
+    @Test
+    public void writeStreamAsZip() throws IOException {
+        class AnonymousClass implements EntryVersionHelper {
+            @Override
+            public EntryDAO getDAO() {
+                return null;
+            }
+        }
+        AnonymousClass anonymousClass = new AnonymousClass();
```

Comment:
Since you're not reusing `class AnonymousClass` this can be simpler as something like the following
```
    AnonymousClass anonymousClass = new EntryVersionHelper() {
            @Override
            public EntryDAO getDAO() {
                return null;
            }
    }
```

Reply:
Interesting `EntryVersionHelper anonymousClass = () -> null;`